python/TestHarness/unit_tests/test_HarnessTester.py

```diff
@@ -44,8 +44,8 @@ def testDiffs(self):
             self.runTests('-i', 'diffs')
 
         e = cm.exception
-        self.assertRegexpMatches(e.output, 'test_harness\.exodiff.*?FAILED \(EXODIFF\)')
-        self.assertRegexpMatches(e.output, 'test_harness\.csvdiff.*?FAILED \(CSVDIFF\)')
+        self.assertRegexpMatches(e.output, 'test_harness\.exodiff.*?Running exodiff')
+        self.assertRegexpMatches(e.output, 'test_harness\.csvdiff.*?Running CSVDiffer.py')
```

Comment:
You don't want to replace what's here or you'll break what this was testing. (e.g. The fact that it prints FAILED (EXODIFF)). Instead you'll want to add theses lines to check that it's recognizing some sort of content in the middle AND the final message that it failed. Nothings stopping you from having both RegExs.